### PR TITLE
Add support for dl360 HPE

### PR DESCRIPTION
**Pull request by asyedham**
Add rhel8_interfaces for 6048r

### CODE REVIEW

File: tasks/install_os.yml [hunk 1]
```diff
@@ -61,15 +61,15 @@
...
     - name: set hypervisor host to PXE boot (Supermicro)
       shell: ipmitool -I lanplus -H mgmt-{{ hypervisor_host }} -U quads -P {{ chassis_password }} chassis bootdev pxe options=persistent
-      when: vendor is defined and vendor == "supermicro"
+      when:  vendor is defined and (vendor == "supermicro" or vendor == "hpe")
       delegate_to: "{{ hammer_host }}"
       vars:
         ansible_user: root
         ansible_python_interpreter: "{{ hammer_python_interpreter }}"
 
     - name: power cycle hypervisor host (Supermicro)
```
Comment (pradiptapks) [inline tasks/install_os.yml]: Please include HPE in task details

File: tasks/install_os.yml [hunk 2]
```diff
@@ -61,15 +61,15 @@
 
     - name: set hypervisor host to PXE boot (Supermicro)
```
Comment (pradiptapks) [inline tasks/install_os.yml]: Please include HPE in task details

File: tasks/install_os.yml [hunk 3]
```diff
@@ -59,17 +59,17 @@
         ansible_user: root
         ansible_python_interpreter: "{{ hammer_python_interpreter }}"
 
-    - name: set hypervisor host to PXE boot (Supermicro)
+    - name: set hypervisor host to PXE boot (Supermicro or HPE)
       shell: ipmitool -I lanplus -H mgmt-{{ hypervisor_host }} -U quads -P {{ chassis_password }} chassis bootdev pxe options=persistent
-      when: vendor is defined and vendor == "supermicro"
+      when:  vendor is defined and (vendor == "supermicro" or vendor == "hpe")
```
Comment (masco) [inline tasks/install_os.yml]: NIT: vendor in ["supermicro", "hpe"]

File: tasks/install_os.yml [hunk 4]
```diff
@@ -59,17 +59,17 @@
...
       delegate_to: "{{ hammer_host }}"
       vars:
         ansible_user: root
         ansible_python_interpreter: "{{ hammer_python_interpreter }}"
 
-    - name: power cycle hypervisor host (Supermicro)
+    - name: power cycle hypervisor host (Supermicro or HPE)
       shell: ipmitool -I lanplus -H mgmt-{{ hypervisor_host }} -U quads -P {{ chassis_password }} chassis power cycle
-      when: vendor is defined and vendor == "supermicro"
+      when:  vendor is defined and (vendor == "supermicro" or vendor == "hpe")
```
Comment (masco) [inline tasks/install_os.yml]: same here

File: tasks/install_os.yml (hunk 3 above)
Comment (asyedham) [reply tasks/install_os.yml]: done

**Comment by pradiptapks** [comment]
@asyedham One of dl360 node has hw issue. I will review this PR in another lab and will update the status soon

**Comment by pradiptapks** [comment]
@asyedham The profile tagging task is still failed while enabling composable roles in Jetpack environment file.

I tried for 1029u, dl360, r640.

>> Jetpack failures
```
TASK [Setting profile for node type] *********************************************************************************************************************************************************************************************************
Monday 22 March 2021  02:46:42 -0400 (0:00:02.116)       1:37:56.056 ********** 
changed: [undercloud] => (item=1029u) => {"ansible_loop_var": "item", "changed": true, "cmd": "source ~/stackrc\nfor i in $(openstack baremetal node list --format value -c UUID); do\n  if [[ `openstack baremetal node show $i --fields driv
er_info -f json | jq '.driver_info.ipmi_address'` =~ 1029u ]]\n  then\n    openstack baremetal node set $i --property capabilities=profile:baremetal1029u,cpu_vt:true,cpu_hugepages:true,boot_option:local,cpu_txt:true,cpu_aes:true,cpu_hugep
ages_1g:true,boot_mode:bios\n  fi\ndone\n", "delta": "0:00:49.408664", "end": "2021-03-22 06:47:32.456182", "item": "1029u", "rc": 0, "start": "2021-03-22 06:46:43.047518", "stderr": "Error contacting Ironic server: Node ccf92daf-6347-468
2-8e67-2be453fbf1c8 can not be updated while a state transition is in progress. (HTTP 409). Attempt 6 of 6\nNode ccf92daf-6347-4682-8e67-2be453fbf1c8 can not be updated while a state transition is in progress. (HTTP 409)\nError contacting
 Ironic server: Node 3dc85665-d9dc-472f-86b3-fbc2bc9548fd can not be updated while a state transition is in progress. (HTTP 409). Attempt 6 of 6\nNode 3dc85665-d9dc-472f-86b3-fbc2bc9548fd can not be updated while a state transition is in 
progress. (HTTP 409)\nError contacting Ironic server: Node ff4418d3-d300-4533-9ad7-9ae8540377d6 can not be updated while a state transition is in progress. (HTTP 409). Attempt 6 of 6\nNode ff4418d3-d300-4533-9ad7-9ae8540377d6 can not be u
pdated while a state transition is in progress. (HTTP 409)", "stderr_lines": ["Error contacting Ironic server: Node ccf92daf-6347-4682-8e67-2be453fbf1c8 can not be updated while a state transition is in progress. (HTTP 409). Attempt 6 of 
6", "Node ccf92daf-6347-4682-8e67-2be453fbf1c8 can not be updated while a state transition is in progress. (HTTP 409)", "Error contacting Ironic server: Node 3dc85665-d9dc-472f-86b3-fbc2bc9548fd can not be updated while a state transition
 is in progress. (HTTP 409). Attempt 6 of 6", "Node 3dc85665-d9dc-472f-86b3-fbc2bc9548fd can not be updated while a state transition is in progress. (HTTP 409)", "Error contacting Ironic server: Node ff4418d3-d300-4533-9ad7-9ae8540377d6 c
an not be updated while a state transition is in progress. (HTTP 409). Attempt 6 of 6", "Node ff4418d3-d300-4533-9ad7-9ae8540377d6 can not be updated while a state transition is in progress. (HTTP 409)"], "stdout": "", "stdout_lines": []}
changed: [undercloud] => (item=r640) => {"ansible_loop_var": "item", "changed": true, "cmd": "source ~/stackrc\nfor i in $(openstack baremetal node list --format value -c UUID); do\n  if [[ `openstack baremetal node show $i --fields drive
r_info -f json | jq '.driver_info.ipmi_address'` =~ r640 ]]\n  then\n    openstack baremetal node set $i --property capabilities=profile:baremetalr640,cpu_vt:true,cpu_hugepages:true,boot_option:local,cpu_txt:true,cpu_aes:true,cpu_hugepage
s_1g:true,boot_mode:bios\n  fi\ndone\n", "delta": "0:00:49.183243", "end": "2021-03-22 06:48:22.663525", "item": "r640", "rc": 0, "start": "2021-03-22 06:47:33.480282", "stderr": "Error contacting Ironic server: Node d2fda8ea-1ff4-494b-ac
8b-fe36fceb21f9 can not be updated while a state transition is in progress. (HTTP 409). Attempt 6 of 6\nNode d2fda8ea-1ff4-494b-ac8b-fe36fceb21f9 can not be updated while a state transition is in progress. (HTTP 409)\nError contacting Iro
nic server: Node fb0f4dc0-f61c-46a7-b269-809cb8bb6b5b can not be updated while a state transition is in progress. (HTTP 409). Attempt 6 of 6\nNode fb0f4dc0-f61c-46a7-b269-809cb8bb6b5b can not be updated while a state transition is in prog
ress. (HTTP 409)\nError contacting Ironic server: Node 25bd9925-96b7-40e3-afa3-35c44bdb5cd5 can not be updated while a state transition is in progress. (HTTP 409). Attempt 6 of 6\nNode 25bd9925-96b7-40e3-afa3-35c44bdb5cd5 can not be updat
ed while a state transition is in progress. (HTTP 409)", "stderr_lines": ["Error contacting Ironic server: Node d2fda8ea-1ff4-494b-ac8b-fe36fceb21f9 can not be updated while a state transition is in progress. (HTTP 409). Attempt 6 of 6", 
"Node d2fda8ea-1ff4-494b-ac8b-fe36fceb21f9 can not be updated while a state transition is in progress. (HTTP 409)", "Error contacting Ironic server: Node fb0f4dc0-f61c-46a7-b269-809cb8bb6b5b can not be updated while a state transition is 
in progress. (HTTP 409). Attempt 6 of 6", "Node fb0f4dc0-f61c-46a7-b269-809cb8bb6b5b can not be updated while a state transition is in progress. (HTTP 409)", "Error contacting Ironic server: Node 25bd9925-96b7-40e3-afa3-35c44bdb5cd5 can n
ot be updated while a state transition is in progress. (HTTP 409). Attempt 6 of 6", "Node 25bd9925-96b7-40e3-afa3-35c44bdb5cd5 can not be updated while a state transition is in progress. (HTTP 409)"], "stdout": "", "stdout_lines": []}
failed: [undercloud] (item=dl360) => {"ansible_loop_var": "item", "changed": true, "cmd": "source ~/stackrc\nfor i in $(openstack baremetal node list --format value -c UUID); do\n  if [[ `openstack baremetal node show $i --fields driver_i
nfo -f json | jq '.driver_info.ipmi_address'` =~ dl360 ]]\n  then\n    openstack baremetal node set $i --property capabilities=profile:baremetaldl360,cpu_vt:true,cpu_hugepages:true,boot_option:local,cpu_txt:true,cpu_aes:true,cpu_hugepages
_1g:true,boot_mode:bios\n  fi\ndone\n", "delta": "0:00:37.664664", "end": "2021-03-22 06:49:01.372085", "item": "dl360", "msg": "non-zero return code", "rc": 1, "start": "2021-03-22 06:48:23.707421", "stderr": "Error contacting Ironic ser
ver: Node 6316a409-2b83-429d-be3b-39ee3084dc2e can not be updated while a state transition is in progress. (HTTP 409). Attempt 6 of 6\nNode 6316a409-2b83-429d-be3b-39ee3084dc2e can not be updated while a state transition is in progress. (
HTTP 409)\nError contacting Ironic server: Node 399c77ed-855a-4c19-b8d5-da6a0867aa10 can not be updated while a state transition is in progress. (HTTP 409). Attempt 6 of 6\nNode 399c77ed-855a-4c19-b8d5-da6a0867aa10 can not be updated whil
e a state transition is in progress. (HTTP 409)", "stderr_lines": ["Error contacting Ironic server: Node 6316a409-2b83-429d-be3b-39ee3084dc2e can not be updated while a state transition is in progress. (HTTP 409). Attempt 6 of 6", "Node 6
316a409-2b83-429d-be3b-39ee3084dc2e can not be updated while a state transition is in progress. (HTTP 409)", "Error contacting Ironic server: Node 399c77ed-855a-4c19-b8d5-da6a0867aa10 can not be updated while a state transition is in prog
ress. (HTTP 409). Attempt 6 of 6", "Node 399c77ed-855a-4c19-b8d5-da6a0867aa10 can not be updated while a state transition is in progress. (HTTP 409)"], "stdout": "", "stdout_lines": []}
```

Flavor:
```
$ openstack flavor list| grep baremetal
| 4380b051-6816-4865-866a-c2d597113472 | baremetal1029u | 4096 |   40 |         0 |     1 | True      |
| 5581386e-684e-4381-93ac-7516d4bdbc06 | baremetal      | 4096 |   40 |         0 |     1 | True      |
| e5665030-aa5d-4c14-b1e9-2baa8deb5172 | baremetaldl360 | 4096 |   40 |         0 |     1 | True      |
| fce30c89-4e34-4c5e-9480-f1e2affefb62 | baremetalr640  | 4096 |   40 |         0 |     1 | True      |
```

overcloud profiles:
```
$ openstack overcloud profiles list
+--------------------------------------+-----------+-----------------+-----------------+-------------------+
| Node UUID                            | Node Name | Provision State | Current Profile | Possible Profiles |
+--------------------------------------+-----------+-----------------+-----------------+-------------------+
| ccf92daf-6347-4682-8e67-2be453fbf1c8 |           | available       | None            |                   |
| 3dc85665-d9dc-472f-86b3-fbc2bc9548fd |           | available       | None            |                   |
| ff4418d3-d300-4533-9ad7-9ae8540377d6 |           | available       | None            |                   |
| d2fda8ea-1ff4-494b-ac8b-fe36fceb21f9 |           | available       | None            |                   |
| fb0f4dc0-f61c-46a7-b269-809cb8bb6b5b |           | available       | None            |                   |
| 25bd9925-96b7-40e3-afa3-35c44bdb5cd5 |           | available       | None            |                   |
| 6316a409-2b83-429d-be3b-39ee3084dc2e |           | available       | None            |                   |
| 399c77ed-855a-4c19-b8d5-da6a0867aa10 |           | available       | None            |                   |
+--------------------------------------+-----------+-----------------+-----------------+-------------------+

```

**Pull request closed without merging**